src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredMale = century
+    ? people.filter(
+      (person) =>
+        Math.ceil(person.died / 100) === century && person.sex === 'm'
+    )
+    : people.filter(
+      (person) => person.sex === 'm'
+    );
```

Comment:
you can put the checks inside `filter` method
```suggestion
  const filteredMale = people.filter((person) => person.sex === 'm' &&
    century ? Math.ceil(person.died / 100) === century : true);
```